src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/InputNoWhitespaceBeforeCaseDefaultColon.java

```diff
@@ -0,0 +1,54 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebeforecasedefaultcolon;
+
+/*
+* Config = default
+*/
+public class InputNoWhitespaceBeforeCaseDefaultColon {
+    {
+        switch(1) {
+            case 1 : // violation
+                break;
+            case 2:
+                break;
+            default : // violation
+                break;
+        }
+
+        switch(2) {
+            case 2:
+                break;
+            default:
+                break;
+
+        }
+
+        switch(3) {
+            case 3 /* space around*/ : // violation
+                break;
+            default/* no space around */:
+                break;
+        }
+
+        switch(4) {
+            case 4
+                    : // violation
```

Comment:
please make extra case for: on next line by in good column position relative to previos line:
```
case 4
      : // violation
```

Reply:
Done.